qemu/tests/cfg/qsd_live_snapshot.cfg

```diff
@@ -0,0 +1,37 @@
+- qsd_live_snapshot:
+    type = qsd_live_snapshot
+    start_vm = no
+    images + = "stg1"
```

Comment:
images += " stg1"
note that 1. no space between '+=', 2. there is a space before 'stg1'

Reply:
This is the root cause in your code why stg1 was created twice

Reply:
Updated it to:
 images += " stg1"
 boot_drive_stg1 = no
 boot_drive_sn1 = ${boot_drive_stg1}

Thanks, zhenchao.